integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -640,6 +650,18 @@ case class CarbonLoadDataCommand(
     } else {
       Dataset.ofRows(sparkSession, convertedPlan)
     }
+    try {
+      // Trigger auto compaction
+      CarbonDataRDDFactory.handleSegmentMerging(
+        sparkSession.sqlContext,
+        carbonLoadModel,
+        table,
+        operationContext)
+    } catch {
+      case e: Exception =>
+        throw new Exception(
+          "Dataload is success. Auto-Compaction has failed. Please check logs.")
```

Comment:
include the `e` in the constructor of Exception

Reply:
ok